### ISSUE TITLE
Create a new contact & group on Connections edit screen

### ISSUE THREAD

**Issue by mikeallbutt** [DiscipleTools/disciple-tools-mobile-app#371]
The connections screen in edit mode has lots of fields that can be used to connect the contact with other contacts and groups, but this can only be done if the contact or group has already been created.

It would be helpful to have a quick way to create a new contact or create a new group and then apply them to the appropriate field without having to go and first create them.

**Comment by ahillbilly** [comment]
I agree that this is a great feature.  In my mind it is a non-MVP feature with the work around of exiting the existing contact (A), creating the new contact (B) and then connecting A to B within B's connections tab.
